### ISSUE TITLE
In presence of subcommands, help text does not advertise the presence of options

### ISSUE THREAD

**Issue by Shnatsel** [pacak/bpaf#4]
Right now the help text for `cargo supply-chain` looks like this:

```
Gather author, contributor and publisher data on crates in your dependency graph

Usage: COMMAND

Available options:
    -h, --help      Prints help information
    -v, --version   Prints version information

Available commands:
    publishers  List all crates.io publishers in the depedency graph
    crates      List all crates in dependency graph and crates.io publishers for each
    json        Like 'crates', but in JSON and with more fields for each publisher
    update      Download the latest daily dump from crates.io to speed up other commands
```

I feel that `Usage: COMMAND` is a little misleading: each subcommand accepts a host of arguments, which vary depending on the command in question, but are in no way mentioned in the top-level auto-generated help (they are listed if you call `cargo supply-chain <subcommand> --help`, however).

Would it be possible to reflect the presence of options passed to subcommands, something like `COMMAND [OPTIONS]`?

**Comment by pacak** [comment]
As a quick fix you can take a look at https://docs.rs/bpaf/0.3.2/bpaf/struct.Info.html#method.usage which allows you do just that. I need to check how other systems implement that. I might just replace `COMMAND` token with `COMMAND [...]` since at least `--help` is always available...

**Comment by pacak** [comment]
How do you feel about displaying `COMMAND ...` instead of `COMMAND`? Is that an indication clear enough that you can pass something else?

**Comment by pacak** [comment]
Fixed in 016ab2a
